src/v/model/namespace.h

```diff
@@ -40,7 +40,7 @@ inline const model::topic kafka_consumer_offsets_topic("__consumer_offsets");
 inline const model::topic_namespace kafka_consumer_offsets_nt(
   model::kafka_namespace, kafka_consumer_offsets_topic);
 
-inline const model::topic kafka_audit_logging_topic("__audit_log");
+inline const model::topic kafka_audit_logging_topic("redpanda.audit_log");
```

Comment:
constexpr ?

Reply:
question for denis, do we do underscores or dot notation. i.e.: 

`redpanda.audit.log` or `redpanda.audit_log` or `redpanda.auditlog` 

we should come up with a consistent convetion @deniscoady

Reply:
Dot notation to me feels like a namespace. How about we do `redpanda.audit_log` since I don't think we'd have any other `redpanda.audit.*` topics.

Reply:
`constexpr` breaks the OSS build

Reply:
I do not want to move this forward until we have confirmation on the format.  @emaxerrno @deniscoady

Reply:
Per convo with Matt, `_redpanda.audit_log`